### PR TITLE
Expand v0.9 changelog

### PR DESCRIPTION
**Pull request by galenlynch**

### CODE REVIEW

**Comment by IanButterworth** [comment]
Great to have this all detailed. Thanks!

Do we also want to add a warning and suggestions regarding what to do about https://github.com/JuliaIO/VideoIO.jl/issues/308

**Comment by galenlynch** [comment]
Done. Wording is not the cleanest or most consistent with hyphenations etc so feel free to edit as you see fit.

File: CHANGELOG.md [hunk 1]
```diff
@@ -106,3 +140,21 @@ v0.9.0
...
+v0.9. This is because by default user supplied data is assumed to be full-range
+(jpeg), while the default libav output range is limited range (mpeg), and
+VideoIO will now scale the user data to fit in the limited destination range.
+Prior to v0.9, no such automatic scaling would be done, causing the user data to
+be simply clipped. While this may seem like a regression, it is actually the
+consequence of fixing a bug in the previous versions of VideoIO.
+
+To avoid this slowdown, either specify `color_range=2` in `encoder_options`, or
+alternatively specify the color space of the user-supplied data to already be
+limited range. Note that `color_range=2` may produce videos that are
```
Comment (IanButterworth) [inline CHANGELOG.md:156]: Is there a short tip we could add on what to set to declare it as already limited range?

Comment (galenlynch) [reply CHANGELOG.md:156]: oh sure, I'll add that soon.

File: CHANGELOG.md [hunk 2]
```diff
@@ -106,3 +140,31 @@ v0.9.0
...
+incompatible with some video players.
+
+If you are encoding data that is already limited range, then the simplest way to
+avoid automatic scaling is to indicate that the user data is in the FFmpeg
+default colorspace. This is accomplished by setting
+`input_colorspace_details = VideoIO.VioColorspaceDetails()` when encoding the
+video. While the FFmpeg default color range is "unknown", settings this will
```
Comment (galenlynch) [inline CHANGELOG.md:163]: typo, "settings" should be "setting"

Comment (IanButterworth) [reply CHANGELOG.md:163]: fixed on master

**Pull request merged**